src/app/components/discover/discover.component.css

```diff
@@ -0,0 +1,3 @@
+.top-space{
+    margin-top: 20px;
```

Comment:
There are bootstrap utility functions for adding margins and paddings - might be curious to check out :) 
https://getbootstrap.com/docs/4.1/utilities/spacing/